arm_controller:
  type: effort_controllers/JointTrajectoryController
  topic: "test"
  joints:
     - ur5_arm_shoulder_pan_joint
     - ur5_arm_shoulder_lift_joint
     - ur5_arm_elbow_joint
     - ur5_arm_wrist_1_joint
     - ur5_arm_wrist_2_joint
     - ur5_arm_wrist_3_joint
  gains:
    ur5_arm_shoulder_pan_joint: {p: 4000.0, i: 1000.0, d: 100.0, i_clamp_max: 10.0, i_clamp_min: -10.0}
    ur5_arm_shoulder_lift_joint: {p: 2000.0, i: 500.0, d: 100.0, i_clamp_max: 10.0, i_clamp_min: -10.0}
    ur5_arm_elbow_joint: {p: 1500.0, i: 500.0, d: 50.0, i_clamp_max: 10.0, i_clamp_min: -10.0}
    ur5_arm_wrist_1_joint: {p: 700.0, i: 200.0, d: 10.0, i_clamp: 0.0}
    ur5_arm_wrist_2_joint: {p: 700.0, i: 200.0, d: 10.0, i_clamp: 0.0}
    ur5_arm_wrist_3_joint: {p: 700.0, i: 200.0, d: 10.0, i_clamp: 0.0}
  constraints:
      goal_time: 0.6
      stopped_velocity_tolerance: 0.5
      ur5_arm_shoulder_pan_joint: {trajectory: 0.1, goal: 0.1}
      ur5_arm_shoulder_lift_joint: {trajectory: 0.1, goal: 0.1}
      ur5_arm_elbow_joint: {trajectory: 0.1, goal: 0.1}
      ur5_arm_wrist_1_joint: {trajectory: 0.1, goal: 0.1}
      ur5_arm_wrist_2_joint: {trajectory: 0.1, goal: 0.1}
      ur5_arm_wrist_3_joint: {trajectory: 0.1, goal: 0.1}
  stop_trajectory_duration: 0.5
  state_publish_rate:  25
  action_monitor_rate: 10
